.github/workflows/cifuzz.yml

```diff
@@ -24,7 +24,7 @@ jobs:
   Fuzzing:
     # FIXME: Figure out why 32-bit applications fail to run in docker on Ubuntu 24.04.
     runs-on: ubuntu-22.04
-    if: github.repository == 'systemd/systemd'
+    if: github.repository == 'systemd/systemd' || github.repository == 'systemd/systemd-stable'
```

Comment:
CIFuzz isn't suitable for testing forks.

Reply:
it literally doesn't work with forks: https://github.com/google/oss-fuzz/issues/3731

Reply:
OK. Dropped.